tools/templates/moar/Makefile.in

```diff
@@ -345,6 +345,7 @@ m-install: m-all @@script(install-core-dist.p6)@@ $(SETTING_MOAR)
 	@nfpq($(BASE_DIR)/$(M_BAT_RUNNER))@ @shquot(@script(upgrade-repository.p6)@)@ @nfpq($(DESTDIR)$(PERL6_HOME)/vendor)@
 	@nfpq($(BASE_DIR)/$(M_BAT_RUNNER))@ @shquot(@script(upgrade-repository.p6)@)@ @nfpq($(DESTDIR)$(PERL6_HOME)/site)@
 	@nfpq($(BASE_DIR)/$(M_BAT_RUNNER))@ @shquot(@script(install-core-dist.p6)@)@ @q($(DESTDIR)$(PERL6_HOME))@
+@clean_old_p6_libs@
```

Comment:
`@expand(@m_install@)@` must immediately follow `@clean_old_p6_libs@` on the same line. In the current variant you'll get an empty line here. If we ever feel adventurous to use `make.nqp` it wouldn't be happy about it.

Reply:
Actually there is no empty line there because I chomped of the last \n in the variable. Not sure what of the two is the cleaner solution though...

Reply:
Don't chomp and make it one line. You `clean_old_p6_libs` remains empty you _will_ have an empty line. ;)

Reply:
Done.